bindings/pybind11/idyntree_core.cpp

```diff
@@ -179,89 +50,64 @@ void transformClassDefinition(py::class_<Transform>& transform) {
 }  // namespace
 
 void iDynTreeCoreBindings(pybind11::module& module) {
-  // Vectors and matrices.
-  py::class_<VectorDynSize> vector_dyn(module, "VectorDynSize",
-                                       py::buffer_protocol());
-  vectorDynSizeClassDefinition(vector_dyn);
-  createFixSizeVector<3>(module, "Vector3");
-  createFixSizeVector<4>(module, "Vector4");
-  createFixSizeVector<6>(module, "Vector6");
-
-  py::class_<MatrixDynSize> matrix_dyn(module, "MatrixDynSize",
-                                       py::buffer_protocol());
-  matrixDynSizeClassDefinition(matrix_dyn);
-  createFixSizeMatrix<3, 3>(module, "Matrix3x3");
-  createFixSizeMatrix<4, 4>(module, "Matrix4x4");
-  createFixSizeMatrix<6, 6>(module, "Matrix6x6");
-
-  // Positions, Rotations and Transforms.
-  py::class_<PositionRaw, VectorFixSize<3>>(module, "_PositionRaw")
-      // Do not expose constructor as we do not want users to use this class.
-      .def("__repr__", &PositionRaw::toString);
-
-  py::class_<Position, PositionRaw>(module, "Position")
```

Comment:
I would argue as there is a Rotation class we should keep the position class.

To avoid the pain of calling the constructor with 3 doubles you can also define a constructor that takes a `VectorFixSize<3>` which should be converted automatically from NumPy

Reply:
I restored the class as you suggested